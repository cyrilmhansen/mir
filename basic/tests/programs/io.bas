10 INPUT A
20 PRINT A
30 INPUT "N? ";B
40 PRINT B
50 END
