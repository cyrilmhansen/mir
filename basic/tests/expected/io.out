4
N? 5
